src/screens/ReceiveFunds/03-Confirmation.js

```diff
@@ -157,6 +158,7 @@ export default function ReceiveConfirmation({ navigation, route }: Props) {
     const device = route.params.device;
 
     if (device && !verified) {
+      setAllowNavigation(false);
```

Comment:
@jackthta Can you please explain this change? Is it mandatory for Hedera?

Reply:
My mistake 😅, it probably sneaked through when rebasing. It has been removed 👍🏻.